Analog in (#23)
hostmobility/documentation: docs/interfaces/c61/io.md

---
title: IOs (C61)
tags:
  - ADC
  - Digital IO
  - C61
---

## Overview

> This product has two analog inputs that is combined with two digital outputs available and two digital inputs on the same two connector pins.
> This analog in can be used to detect short circuit or to just monitor some signal between the supported voltage range of the platform. 

Reviewer: ```suggestion
This platform has two analog inputs that are combined with two digital outputs on the same connector pins.
Analog In can be used to detect short-circuits or to monitor certain signals within the supported voltage range of the platform.
```
